docs_src/src/pages/documentation/example_app/openapi.mdx

```diff
@@ -19,13 +19,13 @@ python app.py --disable-openapi
 
 ## How to use?
 
-- Query Params: The typing for query params can be added as `def get(r: Request, query_params=GetRequestParams)` where `GetRequestParams` is a `TypedDict`
+- Query Params: The typing for query params can be added as `def get(r: Request, query_params=GetRequestParams)` where `GetRequestParams` is a subclass of `RequestQuery`
 - Path Params are defaulted to string type (ref: https://en.wikipedia.org/wiki/Query_string)
```

Comment:
maybe we might need to break the norm here at some point in the future.

Reply:
didn't get you? 🤔